src/main/java/com/movie_collection/gui/models/IMovieModel.java

```diff
@@ -6,14 +6,16 @@
 import java.sql.SQLException;
 
 public interface IMovieModel {
-    
+
     ObservableList<Movie> getAllMovies() throws SQLException;
-    ObservableList<Movie> getAllMoviesInTheCategory(int categoryId) throws SQLException;
+    void getAllMoviesInTheCategory(int categoryId) throws SQLException;
```

Comment:
its interesting to have it as a void we can discuss it later when we have the mapper implementation